frontend/src/hooks/guards/useNotOnboarded.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react';
+import useCheckNotOnboarded from './useCheckNotOnboarded';
+
+const useNotOnboarded = () => {
+  const checkNotOnboarded = useCheckNotOnboarded();
+
+  useEffect(() => {
+    const callCheckNotOnboarded = async () => {
+      try {
+        await checkNotOnboarded();
+      } catch (error) {}
```

Comment:
empty catch here is justified since it is used to stop error propagation. im silencing the error since it is not the hook's responsibility to deal with the error. 

- onNotOnboarded: to auto call checkNotOnboarded on mount 
- checkNotOnboarded: check if the user is not onboarded. if he is onboarded then notify using snackbar and redirect

Here, im saying: im not gonna stop any rejected promise but im not handling them as well (but not sure if this is the best practice)